packages/react-log-viewer/src/LogViewer/examples/LogViewer.md

```diff
@@ -0,0 +1,107 @@
+---
+id: Log viewer
+beta: true
+section: components
+propComponents: [LogViewer]
```

Comment:
You should add `LogViewerCustomControl` to `propComponents`. The components that the consumer will be using should be listed here so the docs contain all the props and comments associated with all props so they know how yo use them.